Libraries/D3D12RaytracingFallback/src/FallbackLayer.vcxproj

```diff
@@ -105,13 +105,18 @@ PrebuildCheck.bat</Command>
     </PreBuildEvent>
     <ClCompile>
       <AdditionalIncludeDirectories Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">..Core/;%(AdditionalIncludeDirectories);</AdditionalIncludeDirectories>
+      <AdditionalOptions Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">/D"DUMP_UBERSHADER" %(AdditionalOptions)</AdditionalOptions>
```

Comment:
Does this introduce noticeable overhead?

Reply:
Since this is in the readme to be added, I suppose it shouldn't be set by default

Reply:
Oops. Meant to take that out. Thanks for the find.